qadence_protocols/mitigations/twirl.py

```diff
@@ -1,24 +1,30 @@
 from __future__ import annotations
 
 import itertools
+from collections import Counter
 
 import torch
 
 # import pytest
 from qadence import QuantumCircuit, QuantumModel, block_to_tensor, chain
+from qadence.blocks.utils import unroll_block_with_scaling
 from qadence.noise.protocols import Noise
-from qadence.operations import I, X, Z, kron
+from qadence.operations import RX, RY, I, X, Z, kron
 from qadence.types import BackendName
 from torch import tensor
 
+from qadence_protocols.measurements import Measurements
+
 
 def twirl_swap(n_qubits: int, twirl: tuple, samples_twirl: dict) -> dict:
     output = {}
     operand = "".join(map(str, [1 if i in twirl else 0 for i in range(n_qubits)]))
     for key in samples_twirl.keys():
-        output["{0:b}".format(int(key, 2) ^ int(operand, 2))] = samples_twirl[key]
+        output[
+            ("{:0" + str(n_qubits) + "b}").format(int(key, 2) ^ int(operand, 2))
```

Comment:
I think this can be recast into a syntax that avoids `.format`. Also, I would like to see it tested.

Reply:
The expected results and inputs are not straightforward to be hard coded for the function, im not sure how to do this

Reply:
replaced format with bin, but it also uses a certain formating "0b0111" implicity and I splice it using [2:]